langserver/handler.go

```diff
@@ -198,6 +199,13 @@ func (h *LangHandler) Handle(ctx context.Context, conn JSONRPC2Conn, req *jsonrp
 		}
 		return h.handleSymbol(ctx, conn, req, params)
 
+	case "workspace/reference":
+		var params lspext.WorkspaceReferenceParams
+		if err := json.Unmarshal(*req.Params, &params); err != nil {
```

Comment:
I see this is not handled in the other cases for some reason, but `req.Params` can be nil, in which case this would cause a panic.